### ISSUE TITLE
Processing Video not working with MacOS Catalina (10.15.4)

### ISSUE THREAD

**Issue by leifmarcus** [processing/processing-video#149]
Hello, i tried to run a sketch with included video Capture, but nothing happens.

What i Tried is to use the `r6-v2.0-beta4` release and also updated to the current GStreamer `1.16.2` which is also printed into the Processing console as follows
```
Processing video library using GStreamer 1.16.2
```

When I try to run the example sketch `AsciiVideo` i get the following error:

```
BaseSrc: [avfvideosrc0] : Device video access permission has just been denied
```
Seems like the Implementation that is mentioned [here](https://developer.apple.com/documentation/avfoundation/cameras_and_media_capture/requesting_authorization_for_media_capture_on_macos?language=objc) is missing. Is there a way of solving the Issue anytime soon? I found that many people seem to have this problem with MacOS Catalina.

**Comment by mangolas** [comment]
Just stumbled on this in general trying to use GStreamer command line on Catalina, so even thought I'm not using `processing-video` myself, maybe it could help or give some directions.

I was first trying to open streaming video from my webcam and got exactly same denied error from my command:
`gst-launch-1.0 -vv avfvideosrc ! videoconvert ! x264enc speed-preset=superfast tune=zerolatency pass=qual ! rtph264pay ! udpsink host=127.0.0.1 port=5050 sync=false`

And no dialog for granting permission was show.

But when I just tried to show video in a window, macOS *did* prompt a dialog and asked if Terminal could be allowed to use camera, I did so, and now the above headless video usage is allowed as well. And whole Terminal app is now allowed to use Camera...

With this command the dialog appeared:
`gst-launch-1.0 -vv  avfvideosrc ! video/x-raw,width=1280,height=720 ! autovideosink`
